content/documentation/getting-started/index.adoc

```diff
@@ -52,8 +53,11 @@ oc login -u system:admin
 
 === Install Istio
 
-Make sure your cluster environment has Istio installed.
-For details on how to install Istio, see the link:https://istio.io/docs/setup/[Istio Setup docs].
+Make sure your cluster environment has Istio installed. For details on how to install Istio, see the link:https://istio.io/docs/setup/[Istio Setup docs].  If you are running on OpenShift and prefer Maistra, see link:https://maistra.io/docs/getting_started/install/[Maistra Setup docs].
+
+icon:lightbulb[size=1x]{nbsp}Both Istio and Maistra include Kiali distributions.
+
+If the packaged version of Kiali is sufficient then choose the options to install Kiali and then continue to link:#_open_the_ui[Open the UI].  If you prefer to install the latest Kiali then do not choose the options to install Kiali, or delete it as necessary, and continue with the steps below.
```

Comment:
This isn't clear you are talking about the packaged version **that comes with Istio or Maistra**. Say something like:

```
If the Kiali version that comes packaged with Istio or Maistra is sufficient for you, then you
may install Kiali using the Istio or Maistra installation procedures and then continue to
link:#_open_the_ui[Open the UI]. If you prefer to install the latest Kiali then do not have
Istio or Maistra install Kiali and continue with the steps below. Note that the Kiali installation
script documented below will check to see if Kiali is already installed and, if it is, will ask
to delete it.
```

Reply:
I updated the text